src/store/reducers/ErrorReducer.js

```diff
@@ -1,8 +1,14 @@
-import { LOGIN_ERROR, REGISTER_ERROR } from '../actions/ActionTypes';
+import { LOGIN_ERROR, REGISTER_ERROR , ADD_MOVIE_ERROR} from '../actions/ActionTypes';
 
 const initialState = {
   loginError: false,
-  registerError: false
+  registerError: false,
+  addMovieError: {
+    title : "",
+    description : "",
+    image_url: "",
+    genre_id: ""
+  }
```

Comment:
cemu ovo sluzi ?

Reply:
Prikaze gresku koja se vraca sa bekenda ukoliko validacija nije uspesna. Za svako polje koje korisnik nije popunio pravilno, pise gde je pogresio. Jel se to radi drugacije?

Reply:
da , uglavnom sve greske su u obliku 
```
{
status: 500|422|400|401
message: Bad Request|Unauthroized|....
}
```

A na frontu ces ti isto imati validaciju gdje ces napraviti sta je mandatory ili odredjenog tipa il stav vec i onda ces prikazivati greske odmah prije nego se sibne request